tidb-cloud/branch-manage.md

```diff
@@ -24,8 +24,10 @@ To create a branch, perform the following steps:
 
 1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, and then click the name of your target TiDB Cloud Serverless cluster to go to its overview page.
 2. Click **Branches** in the left navigation pane.
-3. Click **Create Branch** in the upper-right corner.
-4. Enter the branch name, and then click **Create**.
+3. Click on **Create Branch** in the top right corner or click on **Create Branch** in the **...** in the **Action** column in the row of your target parent branch
+4. Enter the branch **Name**.
+5. Select the **Parent branch**; `main` represents the current cluster.
+6. Select the **Include data up to**. 
```

Comment:
```suggestion
4. In the **Create Branch** dialog, configure the following options:

    - **Name**: enter a name for the branch.
    - **Parent branch**: select the original cluster. `main` represents the current cluster.
    - **Include data up to**: choose one of the following:
        - **Current point in time**: create a branch from the current state.
        - **Specific date and time**: create a branch from a specified time.

5. Click **Create**.
```

Reply:
> **Parent branch**: select the original cluster. `main` represents the current cluster.
select the original cluster or a existed branch.

Reply:
Updated

> - **Parent branch**: select the original cluster or an existing branch. `main` represents the current cluster.